examples/localization/src/config.ts

```diff
@@ -1,5 +1,5 @@
 import { collection, config, fields } from '@keystatic/core';
-import { locales } from '../../../packages/keystatic/src/app/l10n/locales';
+import { locales } from '@keystatic/core/src/app/l10n/locales';
```

Comment:
This weird import was intentional because `@keystatic/core/src/app/l10n/locales` won't work because of the `exports` field